app/controllers/publish/courses/fields/what_you_will_study_controller.rb

```diff
@@ -0,0 +1,59 @@
+# app/controllers/publish/courses/fields/what_you_will_study_controller.rb
+module Publish
+  module Courses
+    module Fields
+      class WhatYouWillStudyController < ApplicationController
+        include CopyCourseContent
+        before_action :authorise_with_pundit
```

Comment:
I think is best not to add the solution to the name of the method. `pundit`, one day if is extremely unlikely we might change the way we authorise users. I'd suggest rename to authorise_provider or authorise_user or similar :)